components/camera/camera.go

```diff
@@ -1,5 +1,3 @@
-//go:build !no_cgo
```

Comment:
[note] Confirmed I am able to import `camera` package in a module with no cgo issues.